tests/test_pretty.py

```diff
@@ -0,0 +1,29 @@
+import unittest
+from nelox.pretty_printer import pretty
+from nelox.Expr import Literal, Variable, List
+from nelox.token_type import TokenType
+from nelox.nelox_token import Token
+
+def make_token(type_, lexeme):
+    return Token(type_, lexeme, None, 0)
+
+class PrettyPrinterTest(unittest.TestCase):
+    def test_literal(self):
+        self.assertEqual(pretty(Literal(42)), "42")
+
+    def test_variable(self):
+        token = make_token(TokenType.IDENTIFIER, "x")
```

Comment:
You might want to make a PR that unwraps tokens in the parser, so that you can just use a string instead of having to make all these tokens.

Reply:
That's fine, please open an issue for now